lightning/src/ln/channelmanager.rs

```diff
@@ -1407,24 +1407,11 @@ pub struct ChannelDetails {
 	///
 	/// This value will be `None` for objects serialized with LDK versions prior to 0.0.115.
 	pub feerate_sat_per_1000_weight: Option<u32>,
-	/// Our total balance.  This is the amount we would get if we close the channel.
```

Comment:
Could add a note at the top level of this struct with a reference to the `claimable_balances` method.